test/browser/createElementProduction.test.js

```diff
@@ -0,0 +1,10 @@
+import { h } from '../../dist/preact';
+
+describe('createElement production build', () => {
+	it('should produce a correct structure', () => {
+		const el = h('div', null, h('span', null), h('br', null));
+		expect(JSON.stringify(el)).to.equal(
+			'{"type":"div","props":{"children":[{"type":"span","props":{},"__k":null,"__p":null,"__b":0,"__e":null,"l":null,"__c":null},{"type":"br","props":{},"__k":null,"__p":null,"__b":0,"__e":null,"l":null,"__c":null}]},"__k":null,"__p":null,"__b":0,"__e":null,"l":null,"__c":null}'
```

Comment:
Instead of checking against the shape directly, we could just check if `JSON.stringify` doesn't throw an error. According to [the spec](https://tc39.es/ecma262/#sec-json.stringify) it's must throw a `TypeError` when it encounters a cyclic structure.

Reply:
That's really smart 😁